mscore/continuouspanel.cpp

```diff
@@ -276,20 +277,20 @@ void ContinuousPanel::paint(const QRect&, QPainter& painter)
       painter.setBrush(preferences.getColor(PREF_UI_CANVAS_FG_COLOR));
       QRectF bg(_rect);
 
-      bg.setWidth(_widthClef + _widthKeySig + _widthTimeSig + _leftMarginTotal + _panelRightPadding);
+      painter.setClipRect(_rect);
+      painter.setClipping(true);
+
       QPixmap* fgPixmap = _sv->fgPixmap();
-      if (fgPixmap == 0 || fgPixmap->isNull())
+      if (fgPixmap == 0 || fgPixmap->isNull()) {
             painter.fillRect(bg, preferences.getColor(PREF_UI_CANVAS_FG_COLOR));
+            }
       else {
-            painter.setMatrixEnabled(false);
+            painter.setMatrixEnabled(false);            // Paints background when custom backing is used
             painter.drawTiledPixmap(bg, *fgPixmap, bg.topLeft()
                - QPoint(lrint(_sv->matrix().dx()), lrint(_sv->matrix().dy())));
             painter.setMatrixEnabled(true);
```

Comment:
Instead of all these changes I found it is enough to remove the lines 284/288 and 287/291 to solve issue 279877.